fbfmaproom/assets/style.css

```diff
@@ -92,6 +92,15 @@ table.supertable tr th:first-child {
     color: rgb(67, 104, 176);
 }
 
+table.supertable thead tr:first-child th {
+    background-color: rgb(241, 241, 241);
+    border-right-style: none;
+}
+
```

Comment:
did you make sure a check these changes don't produce any visual glitches? they can be subtle and surprising

Reply:
I did find and fix some glitches, but it would be good to get more eyes on it. It's running on http://shortfin01.iri.columbia.edu:9999/fbfmaproom/niger